samples/IdentitySample.Mvc/Startup.cs

```diff
@@ -82,9 +67,21 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
             app.UseErrorPage()
                .UseStaticFiles()
                .UseIdentity()
-               .UseFacebookAuthentication()
-               .UseGoogleAuthentication()
-               .UseTwitterAuthentication()
+               .UseFacebookAuthentication(options =>
```

Comment:
Note this change is optional now, either way works (via ConfigureServices or via Use)